src/PhpSpreadsheet/Writer/Xlsx/Rels.php

```diff
@@ -295,12 +295,25 @@ public function writeDrawingRelationships(\PhpOffice\PhpSpreadsheet\Worksheet\Wo
             if ($iterator->current() instanceof \PhpOffice\PhpSpreadsheet\Worksheet\Drawing
                 || $iterator->current() instanceof MemoryDrawing) {
                 // Write relationship for image drawing
+                /** @var \PhpOffice\PhpSpreadsheet\Worksheet\Drawing $drawing */
+                $drawing = $iterator->current();
                 $this->writeRelationship(
                     $objWriter,
                     $i,
                     'http://schemas.openxmlformats.org/officeDocument/2006/relationships/image',
-                    '../media/' . str_replace(' ', '', $iterator->current()->getIndexedFilename())
+                    '../media/' . str_replace(' ', '', $drawing->getIndexedFilename())
                 );
+
+                if (!empty($drawing->getHyperlink()->getUrl())) {
+                    ++$i;
+                    $this->writeRelationship(
+                        $objWriter,
+                        $i,
+                        'http://schemas.openxmlformats.org/officeDocument/2006/relationships/hyperlink',
+                        $drawing->getHyperlink()->getUrl(),
+                        $drawing->getHyperlink()->isInternal() ? '' : 'External'
+                    );
+                }
             }
```

Comment:
The `if` and the `?:` here raise the "Condition count" from 8 to 10 and together with the added lines themselves raised the rating from B to C.
[Scrutinizer about Rels::writeDrawingRelationships()](https://scrutinizer-ci.com/g/PHPOffice/PhpSpreadsheet/inspections/4adc3239-9aac-4b9f-9d9f-96ff77f47983/code-structure/operation/PhpOffice%5CPhpSpreadsheet%5CWriter%5CXlsx%5CRels%3A%3AwriteDrawingRelationships).

Reply:
Scrutinizer has some suggestions how to fix this particular issue.

Reply:
Again the entire `if` should go into a new private method (even if it's short). Something like `writeDrawingHyperLink(XMLWriter $objWriter, BaseDrawing $drawing)`